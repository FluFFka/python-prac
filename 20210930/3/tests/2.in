1,5,3,0,4,2,10,5,10,0
8,3,2,2,6,4,0,7,1,2
10,6,6,9,7,7,2,3,4,10
8,1,10,6,5,3,3,10,7,2
2,9,8,9,4,4,6,9,8,6
7,3,8,0,3,1,8,9,3,6
1,6,5,10,1,6,8,8,4,3
1,7,9,8,0,5,7,5,10,0
8,7,10,2,9,6,0,8,4,0
7,4,1,0,6,1,9,4,4,9
4,3,3,2,10,8,0,1,2,1
4,9,0,4,3,10,3,10,10,8
4,10,2,1,4,0,6,4,0,3
0,6,7,4,1,8,9,5,4,0
6,7,1,2,2,4,2,10,7,4
3,2,0,0,10,4,2,1,7,8
3,10,2,8,10,3,10,3,3,8
8,1,4,9,4,0,9,6,5,7
1,10,8,3,3,5,7,0,7,9
10,1,10,0,9,6,0,8,2,3
