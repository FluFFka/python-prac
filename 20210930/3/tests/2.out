[146, 315, 133, 188, 215, 162, 260, 165, 219, 287]
[177, 152, 104, 116, 200, 167, 129, 178, 170, 158]
[285, 334, 260, 155, 369, 354, 238, 320, 283, 263]
[231, 322, 213, 199, 278, 210, 302, 226, 218, 253]
[270, 412, 261, 245, 310, 304, 369, 333, 317, 347]
[252, 276, 176, 194, 306, 177, 247, 226, 174, 247]
[194, 317, 194, 221, 271, 238, 332, 243, 249, 281]
[154, 389, 191, 202, 245, 233, 342, 203, 255, 305]
[240, 322, 131, 164, 265, 230, 229, 272, 267, 270]
[240, 254, 185, 163, 307, 225, 186, 235, 196, 237]
